### PR TITLE
Finish adding unit tests

### PR DESCRIPTION
**Pull request by serban-cristian**

### CODE REVIEW

File: tests/ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests/FunctionalTests.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+using ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Hosting;
+using ES.FX.Ignite.Serilog.Hosting;
+using ES.FX.Ignite.Hosting;
+using Microsoft.AspNetCore.Builder;
+using Seq.Api;
+
+namespace ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests
+{
+
+    public class FunctionalTests(SeqContainerFixture seqFixture)
```
Comment (winromulus) [inline tests/ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests/FunctionalTests.cs]: Convert to SUT. Call endpoint with client and check exported logs

**Review by winromulus — CHANGES_REQUESTED** [review]

File: tests/ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests/FunctionalTests.cs (hunk 1 above)
Comment (serban-cristian) [reply tests/ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests/FunctionalTests.cs]: should be done, ty

**Comment by serban-cristian** [comment]
@winromulus can you review it again, pls?

**Review by winromulus — DISMISSED** [review]
LGTM

**Pull request merged**